server/API DOCS/DELETE.md

```diff
@@ -38,5 +38,26 @@
     * **Code:** `401` <br />
       **Message:** `Authorization Required` <br />
 
+* **Required Filed**
+    * **Auth/ Admin Auth: true/true**
+
+**3. DELETE profile**
+* **URL**
+    `/api/deleteProfile`
+
+* **METHOD**
+    `DELETE`
+
+* **URL params**
+    `none`
+
+* **Success Response**<br />
+    * **Code:** `200` <br />
+      **Message:** `profile deleted` <br />
+
+* **Error Response**
+    * **Code:** `400` <br />
+      **Message:** `Something went wrong` <br />
+
```

Comment:
After this line is counted in section 3.
also Here not need to be admin auth so you can write like this.

Auth/ Admin Auth: true/false (First for user Auth/second for admin auth)